app/src/main/java/protect/card_locker/CardShortcutConfigure.java

```diff
@@ -47,25 +47,19 @@ public void onCreate(Bundle bundle) {
             finish();
         }
 
-        // If all cards are archived, bail
-        if (DBHelper.getArchivedCardsCount(mDatabase) == cardCount) {
-            Toast.makeText(this, R.string.noUnarchivedCardsMessage, Toast.LENGTH_LONG).show();
-            finish();
-        }
-
         final RecyclerView cardList = binding.list;
         GridLayoutManager layoutManager = (GridLayoutManager) cardList.getLayoutManager();
         if (layoutManager != null) {
             layoutManager.setSpanCount(getResources().getInteger(R.integer.main_view_card_columns));
         }
 
-        Cursor cardCursor = DBHelper.getLoyaltyCardCursor(mDatabase, DBHelper.LoyaltyCardArchiveFilter.Unarchived);
-        mAdapter = new LoyaltyCardCursorAdapter(this, cardCursor, this);
+        Cursor cardCursor = DBHelper.getLoyaltyCardCursor(mDatabase, DBHelper.LoyaltyCardArchiveFilter.All);
```

Comment:
This changes the shortcut configuration activity to always show archived cards, which could not be selected before, and without a way to filter them out in the display options. Just making sure that's intentional?

Reply:
I mean it might be nicer to have the whole callback here, but also, it's a rarely used activity. And I figured there was no real valid reason to hide archived cards before. Seems fine to me to show it by default?

Reply:
Yeah, just pointing out it's a change from before, and making sure it's intentional. Though I would kinda assume archived cards are not actively being used, so I think it would be weird to want to create a shortcut to one. But idk.

Reply:
> it's a rarely used activity

Yeah I only realised this existed when reviewing the PR :)

Reply:
I have no strong opinion either way.

Reply:
> I think it would be weird to want to create a shortcut to one

Same, but also, it'd always be at the end of the list and it has its own symbol. So people will see other cards first anyway. I don't expect anyone to use it but see no reason to block them from doing this.